tests/core/test_scripts.py

```diff
@@ -2589,7 +2590,7 @@ def test_do_run(self, db: DatabaseTransactionFixture):
         collection = db.collection(
             name="BiblioBoard Test Collection", settings=settings.dict()
         )
-        collection.libraries = [library]
+        collection.libraries = [library, library2]
```

Comment:
It would be good to include some other collection types here, ones that we don't expect to be picked up like an overdrive collection etc.